src/code/z_scene.c

```diff
@@ -443,8 +443,8 @@ void Scene_CommandAlternateHeaderList(PlayState* play, SceneCmd* cmd) {
 }
 
 void Scene_CommandCutsceneData(PlayState* play, SceneCmd* cmd) {
-    osSyncPrintf("\ngame_play->demo_play.data=[%x]", play->csCtx.segment);
-    play->csCtx.segment = SEGMENTED_TO_VIRTUAL(cmd->cutsceneData.data);
+    osSyncPrintf("\ngame_play->demo_play.data=[%x]", play->csCtx.script);
+    play->csCtx.script = SEGMENTED_TO_VIRTUAL(cmd->cutsceneData.data);
 }
```

Comment:
Should `Scene_CommandCutsceneData` and `cmd->cutsceneData.data` be renamed to involve "script"?

Reply:
it can, but I also see it as a scene/room thing (which absolutely do need more passes)
changing a scene command requires updating a million things (function name, enum name, macro itself, struct, list goes on)
so id rather wait personally lol